docs/rest_api.md

```diff
@@ -88,6 +88,20 @@ Possible errors:
 
 Delete the user whose username is `<username>`
 
+## Projects
+
+### project_add_network
+
+`POST /project/<project>/add_network`
+
+Add a network to project named <project>.
```

Comment:
Could you reflect that you are adding _access_ to a network for this and project_remove_network?